lib/include/cpp-json/json_reader.h

```diff
@@ -3,7 +3,7 @@
 #define JSON_READER_H_
 
 #include <cassert>
-#include <cstddef>
+#include <stddef.h>
```

Comment:
I imagine it wasn't intentional, but i'd like to be consitent regarding `stddef.h` and `cstddef` here.